zeppelin-server/src/test/java/org/apache/zeppelin/rest/ZeppelinSparkClusterTest.java

```diff
@@ -130,6 +130,9 @@ public void pySparkTest() throws IOException {
 //            p.getRepl("org.apache.zeppelin.spark.SparkInterpreter").open();
             note.run(p.getId());
             waitForFinish(p);
+            if (p.getStatus() != Status.FINISHED){
+                LOG.info("pySparkTest: " + p.getResult().toString());
```

Comment:
added this to shed some light on wth is going on and why Pyspark is failing on Travis